packages/test-in-console/test.css

```
@@ -5,4 +5,5 @@
  * makes sure that at least one `.css` file can always be found, when the
  * tests are run.
  */
-.test { }
+.test {
+}
```

Stylelint (17.7.0)

[error] Empty block (block-no-empty)

(block-no-empty)